add script to create council data

Djangofs opened the pull request:
Lets us create test data inside the Azure portal - running the script: `node scripts/create-council-data.js --council wrexham --count 10` for example. This will pull data from Mockaroo API according to the council and number of records, splitting the records between different types (Sole Trader, partnership, company etc) and saving them into the database configured for that environment. It uses the same DB code and config as the application so will always run against the correct environment.

I will invite @Matthew-Moles-Civica and @off-beat to the Mockaroo project

off-beat commented:
In addition, as discussed, the Mockaroo schema was not quite right as at least one field, business other details, was not limited to the 255 characters permitted in the database.

Djangofs commented on scripts/create-council-data.js:
```diff
@@ -0,0 +1,161 @@
+
+  // Split total number of records into 5 numbers to create distribution of data types
+  while (count > 0) {
+    var s = Math.round(Math.random() * count);
```
I think the best solution to this will be to make sure the length of the final array will be the same as dataTypes length by padding with 0's...will make this change

Djangofs commented on scripts/create-council-data.js:
```diff
@@ -0,0 +1,161 @@
+    const response = await model.create(data);
+    return response;
+  } catch (err) {
+    throw err;
```
No not at all...will remove

Djangofs replied:
done

off-beat commented:
I also tried validating it here and got a couple of errors: https://apidevtools.org/swagger-parser/online/

Djangofs commented on scripts/create-council-data.js:
```diff
@@ -0,0 +1,161 @@
+
+  // Get test data for council
+  dataTypes.forEach((type, index) => {
+    dataPromises.push(getTestData(type, council, splitTotal[index]));
```
fixed this

Djangofs commented:
> In addition, as discussed, the Mockaroo schema was not quite right as at least one field, business other details, was not limited to the 255 characters permitted in the database.

I've fixed this by fixing the database schema - business_other_details should be 1500 in length. Have also fixed the declaraton fields being too long which was causing the same issue.
